storage/azurestorage.py

```diff
@@ -52,7 +54,7 @@ def __init__(
         azure_account_name,
         azure_account_key=None,
         sas_token=None,
-        connection_string=None,
+        endpoint_url=None,
```

Comment:
Should this be something like `azure_endpoint_url` since this is azure specific?

Reply:
@bcaton85 I don't think it matters, since this will be under the azure specific storage config:
```
DISTRIBUTED_STORAGE_CONFIG:
  azure:
  - AzureStorage
  - azure_account_key: ...
    azure_account_name: ...
    azure_container: ...
    storage_path: /quayazuredata/quayregistry
    endpoint_url: ...
```